src/components/Tool/FixCommitmentInfo/Contact.tsx

```diff
@@ -79,225 +123,455 @@ const useStyles = makeStyles()(() => ({
+                          </Box>
+                        </Grid>
+                      </Grid>
+                    </Grid>
+                    {!!donations.length && (
+                      <Grid
+                        container
+                        className={classes.donationsTable}
+                        lg={12}
+                      >
+                        {donations.map((donation) => (
+                          <Grid
+                            key={donation.amount.conversionDate}
+                            display="flex"
+                            flexDirection="column"
+                          >
+                            <Box>
+                              <Typography
+                                fontWeight={700}
+                                variant="body2"
+                                data-testid="donationDate"
+                              >
+                                {DateTime.fromISO(
+                                  donation.amount.conversionDate,
+                                )
+                                  .setLocale(locale || 'en')
+                                  .toLocaleString()}
+                              </Typography>
+                            </Box>
+                            <Box>
+                              <Typography
+                                sx={{ textAlign: 'center' }}
+                                variant="body2"
+                                data-testid="donationAmount"
+                              >
+                                {`${donation.amount.amount} ${donation.amount.currency}`}
+                              </Typography>
+                            </Box>
+                          </Grid>
+                        ))}
+                      </Grid>
+                    )}
+                  </Grid>
+                </Card>
+                <Grid item xs={12} md={3} className={classes.buttonGroup}>
+                  <Box className={classes.buttonGroupBox}>
+                    <Box className={classes.buttonTop}>
+                      <Button
+                        type="submit"
+                        variant="contained"
+                        data-testid="confirmButton"
+                        style={{ width: '100%' }}
+                      >
+                        {t('Confirm')}
+                      </Button>
+                    </Box>
+                    <Box className={classes.buttonBottom}>
+                      <Button
+                        variant="contained"
+                        style={{ width: '100%' }}
+                        data-testid="doNotChangeButton"
+                        onClick={() =>
+                          showModal(
+                            modalContact,
+                            t(
+                              `Are you sure you wish to leave {{source}}'s commitment information unchanged?`,
+                              { source: name },
```

Comment:
If a name has a `&` it doesn't format correctly. I've fixed this issue by using a `<Trans>` component.

Reply:
`            <Trans
              defaults="<bold>Source:</bold> {{where}}"
              shouldUnescape
              values={{ where: contact.source }}
              components={{ bold: <strong /> }}
            />`